src/api/app/views/webui/request/beta_show_tabs/_build_status.html.haml

```diff
@@ -7,61 +7,48 @@
-    - if results.present?
-      - previous_repo = nil
-      - results.each do |result|
-        :ruby
-          repository_name = result.repository.tr('.', '_')
-          expanded = repository_expanded?(collapsed_repositories, repository_name, package_name)
-        - if result.repository != previous_repo
-          .d-flex.flex-row.py-1.bg-light.pl-1.pl-sm-2
-            = link_to(word_break(result.repository, 22),
-              package_binaries_path(project: project, package: package, repository: result.repository),
-              title: "Binaries for #{result.repository}")
-            = collapse_link(expanded, package_name, repository_name)
-
-        .collapse{ class: "collapse-#{package_name}-#{repository_name}#{expanded ? ' show' : ''}",
-                   data: { repository: repository_name, main: package_name } }
-          .d-flex.flex-row.flex-wrap.pt-1
-            .repository-state
-              - if result.is_repository_in_db
-                = repository_status_icon(status: result.state)
-              - else
-                %i.fas.fa-clock.text-warning{ title: 'This result is outdated' }
-              %span.ml-1
+  %h5.text-primary.mb-3= package
+  - if results.present?
+    - results_grouped_by_repository = results.group_by(&:repository)
+    - results_grouped_by_repository.each_pair do |repository, results_per_repository|
+      .row.mb-4
+        .col-12.col-sm-3
+          = link_to(word_break(repository, 22),
+            package_binaries_path(project: project, package: package, repository: repository),
+            title: "Binaries for #{repository}")
+        .col-12.col-sm-9
+          - results_per_repository.each do |result|
+            .row.toggle-result-details-parent.mb-1
+              .col-5.col-md-3
                 = result.architecture
-            .build-state.toggle-build-info-parent
-              %i.fa.fa-question-circle.text-info.px-2.pl-lg-1.toggle-build-info{ title: 'Click to keep it open' }
-              = arch_repo_table_cell(result.repository, result.architecture, package, 'code' => result.code, 'details' => result.details)
-            .build-info.mt-1.ml-3.mb-3.mr-3.collapsed
-              .triangle.center
-              .build-info-content
-                %p.py-1= Buildresult.status_description(result.code)
-                %div
+              .col-5.col-md-2
+                = arch_repo_table_cell(result.repository, result.architecture, package, 'code' => result.code, 'details' => result.details)
+              .col-1.col-md-6
+                - if result.is_repository_in_db
+                  = repository_status_icon(status: result.state)
+                  %span.pl-1.d-none.d-md-inline= repository_info(result.state)
+                - else
+                  %i.fas.fa-clock.text-warning
+                  This result is outdated
+              .col-1.text-right
+                %i.fa.fa-question-circle.text-info.toggle-result-details{ title: 'Click to keep it open' }
```

Comment:
This is a bit confusing now. The red parts are about the repository, the green parts are about the package.

![Screenshot from 2022-10-24 11-43-05](https://user-images.githubusercontent.com/514785/197498823-7bd1dafe-9eb5-48a5-abcb-3e35073ec28b.png)

Reply:
Suggestion: The ⚙️ should move next to 15.3 and when clicked expand the explanation (Build jobs exist for this repository) below it. The content of the grey box should move to the place there the ⚙️ is.

Reply:
Do you think re-ordering the content like follows could help?

![small_viewport_build_results](https://user-images.githubusercontent.com/2581944/197503569-24c4d6ce-0d09-4b55-9c53-b6e5363f5e24.png)

![large_viewport_build_results](https://user-images.githubusercontent.com/2581944/197503581-abd70449-6ce4-45fa-a93b-fa1397da3035.png)

This way, the order of what we display from left to right corresponds to what we display from top to bottom in the details box.

Reply:
On small viewports you have no problem. There you have kind of a table. 

![Screenshot from 2022-10-24 12-36-44](https://user-images.githubusercontent.com/514785/197508138-db15484a-0aec-42ca-aba9-b1103db8505d.png)

and clicking the ❓ will expand some "help" about the build/repo status.

Only on large viewports it's gets very confusing as the name of repository, it's state and the explanation of the repositories have no proximity. Same for the builds state and explanation.

Reply:
I'm trying to find an alternative... as the repository state is coupled with the architecture... :thinking: 

![Screenshot 2022-10-24 at 11-54-15 openSUSE build service Server](https://user-images.githubusercontent.com/2581944/197513528-d173892a-349e-4900-9a76-918156dedcc9.png)

So far I'm moving the repository state closer to the repository name.

Reply:
> repository state is coupled with the architecture

The repository state is for *all* architectures.

Reply:
This is obs-server right now:

![Screenshot 2022-10-24 at 13-50-10 openSUSE build service Server](https://user-images.githubusercontent.com/2581944/197519224-30a6aa5b-dad2-4de6-be1d-8674a6290591.png)